mods/d2k/weapons/other.yaml

```diff
@@ -8,35 +8,18 @@ Sound:
 		InaccuracyType: PerCellIncrement
 		Duration: 4 # Has a length of 0c512
 		DamageInterval: 3 # Travels 0c384 between impacts, will hit a target roughly three times
-		Width: 0c512
-		Shape: Flat
-		Falloff: 100, 100, 50
-		Range: 0, 6c0, 11c0
+		Width: 0c756 # in orig d2k width is 2c0 but 100% dmg is only at center and fade out lineary as it come to edges
+		Shape: Cylindrical
+		Falloff: 0, 0, 100, 0 # reflect  original damage
+		Range: 0, 0c450, 4c0, 8c0 # reflect better original damage
```

Comment:
Can you remove those two comments? I don't think they add anything useful for understanding.